Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /tmp/cluster-logging-operator
 COPY vendor/ /tmp/cluster-logging-operator/vendor
 COPY manifests/ /manifests/
 COPY scripts/* /usr/local/bin/scripts/
+COPY files/* /usr/local/bin/files/
```

Comment:
I need to confirm if this fixed the errors i was seeing in the CLO logs when trying to build the curator configmap...